Bottom: c0ec615d4aa8dfd57b560a36e36ca9f4b0a5240f
Top:    5349634cc51fdaf5b2d7c9f4ebe03485b4096ba2
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:09:08 -0500

Refresh of codebase-generator

---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index c14bd7dd..5a366551 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -172,7 +172,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return giturl.owner
+    return '/'.join((giturl.owner, giturl.repo))
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
